swagger.yml

```diff
@@ -42,8 +53,13 @@ definitions:
         description: wall clock time in milliseconds
   CheckResults:
     type: object
-    additionalProperties:
-      $ref: '#/definitions/PodResult'
+    properties:
+      dnsResults:
+        $ref: '#/definitions/DnsResults'
```

Comment:
Since this is a backwards incompatible change, I think we should consider bumping up the major version of the schema. What do you think ?

Reply:
Yep, makes sense